BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.m

```diff
@@ -179,6 +179,7 @@ - (void)performPaginatedRequestWithCached:(BOXItemArrayCompletionBlock)cacheBloc
                                   inRange:(NSRange)range
 {
     BOXFolderPaginatedItemsRequest *paginatedRequest = [[BOXFolderPaginatedItemsRequest alloc] initWithFolderID:self.folderID inRange:range];
+    paginatedRequest.cacheClient = self.cacheClient;
```

Comment:
Yes. `BOXFolderPaginatedItemsRequest` uses a different callback method than this class, `cacheFolderPaginatedItemsRequest:withItems:limit:offset:error:`